src/bindings/python/src/openvino/runtime/utils/data_helpers/wrappers.py

```diff
@@ -26,6 +26,11 @@ def __init__(self, other: InferRequestBase) -> None:
         self._inputs_data = None
         super().__init__(other)
 
+    def _is_single_input(self) -> bool:
+        if len(self.input_tensors) > 1:
```

Comment:
Potentially can be replaced with ``return len(self.input_tensors) == 1`` or ``return len(self.input_tensors) <= 1``?

Reply:
Good catch, pushed.